README.md

```diff
@@ -5765,12 +5765,42 @@ Interface to send syslog data from, e.g. "management".  Valid value is a string.
 
 #### Parameters
 
-##### `name`
-Hostname or address of the Syslog server.  Valid value is a string.
+##### `enable`
+
+Enable or disable syslog logging [true|false].
+
+##### `console`
+
+Console logging severity level [0-7] or 'unset'.
+
+##### `monitor`
+
+Monitor (terminal) logging severity level [0-7] or 'unset'.
+
+##### `source_interface`
+
+Source interface to send syslog data from, for example, "ethernet 2/1" (array of strings for multiple).
 
 ##### `time_stamp_units`
+
 The unit of measurement for log time values.  Valid values are 'seconds' and 'milliseconds'.
 
+##### `vrf`
+
+The VRF associated with source_interface (array of strings for multiple).
+
+##### `logfile_severity_level`
+
+Logfile severity level [0-7]
+
+##### `logfile_name`
+
+Logfile file name to use or 'unset'
+
+##### `logfile_size`
+
+Logging file maximum size [4096-2147483647]

```

Comment:
The range will probably be dynamic on different NX-OS platforms so I would omit the range here.

Reply:
Thanks Mike, removed the range (which simplifies the validation regex hugely! :) )